plugins/module_utils/network/nxos/config/lacp/lacp.py

```diff
@@ -41,14 +41,14 @@ class Lacp(ConfigBase):
     def __init__(self, module):
         super(Lacp, self).__init__(module)
 
-    def get_lacp_facts(self):
+    def get_lacp_facts(self, data=None):
         """ Get the 'facts' (the current configuration)
 
         :rtype: A dictionary
         :returns: The current configuration as a dictionary
         """
         facts, _warnings = Facts(self._module).get_facts(
-            self.gather_subset, self.gather_network_resources
+            self.gather_subset, self.gather_network_resources, data=data
```

Comment:
if the config is a dictionary isn't it should  return empty dict

Reply:
Good catch. It was already there in the existing code, I suppose. But anyway, fixing it.